library/Requests/Transport/cURL.php

```diff
@@ -267,6 +274,7 @@ protected function setup_handle($url, $headers, $data, $options) {
 		curl_setopt($this->fp, CURLOPT_REFERER, $url);
 		curl_setopt($this->fp, CURLOPT_USERAGENT, $options['useragent']);
 		curl_setopt($this->fp, CURLOPT_HTTPHEADER, $headers);
+		curl_setopt($this->fp, CURLOPT_HTTP_VERSION, $options['protocol_version'] == 1.1 ? CURL_HTTP_VERSION_1_1 : CURL_HTTP_VERSION_1_0);
```

Comment:
Strict comparison here please, and it'd be better split into a proper `if` statement instead of the ternary.